charts/gsp-cluster/templates/02-gsp-system/service-operator/role.yaml

```diff
@@ -92,6 +92,7 @@ rules:
   - configmaps
   - events
   - secrets
+  - serviceaccounts
```

Comment:
I don't see a corrosponding "rbac" line in the controller that causes this to get generated?

There should be lines similar to what you can see on the cloudformation.Controller func: https://github.com/alphagov/gsp/blob/master/components/service-operator/internal/aws/cloudformation/controller.go#L128

these ensure that the rbac rules are generated and mean that your changes won't get overwritten in the future